app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+  def update_comment_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
Kindly use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comment_counter` method every time a new comment has been saved to the database, then make the `update_comment_counter` method private :+1: